common/sgx/quote.c

```diff
@@ -567,6 +782,14 @@ oe_result_t oe_verify_quote_with_sgx_endorsements(
     result = OE_OK;
 
 done:
+
+#ifdef OE_BUILD_ENCLAVE
+    if (p_self_target_info)
```

Comment:
No need for the if tests, as these are safe to pass null to, just like free() is.
Removing them reduces code complexity.

Reply:
Make sense, will update code.